server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyFutureServer.scala

```diff
@@ -72,6 +73,7 @@ case class NettyFutureServer(routes: Vector[FutureRoute], options: NettyFutureSe
       NettyBootstrap(
         config,
         new NettyServerHandler(route, unsafeRunAsync, channelGroup, isShuttingDown, config.serverHeader),
+        new ReactiveWebSocketHandler(route, channelGroup, unsafeRunAsync, config.sslContext.isDefined),
```

Comment:
if we were to pass a list of handlers here, maybe we could skip this handler when websockets are not supported? (as in `Future`)